modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/validation/InvalidExtensionConfigTestCase.java

```diff
@@ -55,7 +55,7 @@ public void heisenbergDefaultConfigNegative() throws Exception {
     expectedException.expect(ConfigurationException.class);
     expectedException
         .expectMessage("[validation/heisenberg-default-illegal-config.xml:21]: "
-            + "Element <heisenberg:config> is missing required parameter 'knownAddresses'.");
+            + "Element <heisenberg:config> is missing required parameter 'enemies'.");
```

Comment:
why changing this scenario instead of addding a new one?

Reply:
this test was just an old test with the assertions added at some point. There was no relationship between the name of the test and the expected error.